spec/requests/api/v1/messages/index_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe 'GET /api/v1/conversations/:conversation_id/messages', type: :request do
+  let(:user) { create(:user) }
+  let(:conversation) { create(:conversation, user_id: user.id) }
+
+  before(:each) do
+    conversation.users << user
+  end
```

Comment:
You can use the factory of conversation to avoid this before block
```suggestion
  let(:conversation) { create(:conversation_with_users, user1: user) }
```

Reply:
Thanks! I didn't realized that.